### PR TITLE
Adding CentOS installation directives and warning of only CentOS / Ubuntu Compatibility

### PR DESCRIPTION
**Pull request by bobstrecansky**
Had a need to install this on a CentOS box; wanted to make it easier for others.

### CODE REVIEW

**Comment by luzfcb** [comment]
@bobstrecansky thank you so much for your contribution. Excellent refactoring, simple and accurate.

Said that, I keep having some of the considerations in relation to add another operating system for maintaining.
see the discussion about it here:

https://github.com/pydanny/cookiecutter-django/pull/250#issuecomment-114676903

:bell: @pydanny @jayfk @theskumar @audreyr

**Comment by bobstrecansky** [comment]
@luzfcb no problem!  I am using this for a personal project - thought i'd be able to give back.  If you ever decide to merge this (or if you want help adding support for other OS'es), i'll be happy to help.

File: {{cookiecutter.repo_name}}/install_os_dependencies.sh [hunk 1]
```diff
@@ -51,8 +65,10 @@ function install_or_upgrade()
 
         exit 1
     else
-
-        apt-get update
+        case "$OS" in
+            CentOS) yum update;;
+            Ubuntu) apt-get clean;;
```
Comment (luzfcb) [inline {{cookiecutter.repo_name}}/install_os_dependencies.sh]:
```
            CentOS) yum update;;
            Ubuntu) apt-get update;;
```

**Comment by pydanny** [comment]
Hey @bobstrecansky, thanks so much for the pull request. That said, as @luzfcb said, we're keeping the scope of supporting operating systems small. However, if you want, we can link to your project from ours as a CentOS edition. Seriously, I know people who would really appreciate a supported CentOS friendly fork of cookiecutter-django. :smile:

**Comment by pydanny** [comment]
@bobstrecansky Would you mind emailing me at pydanny@gmail.com? Or contacting me via Gitter? Or Twitter?

**Comment by pydanny** [comment]
If we can find an organization willing to provide someone(s) who can provide ongoing support, we will accept pull requests for Centos support. In the meantime, closing this issue in favor of #561.

**Pull request closed without merging**